xml/book_sles_container.xml

```diff
@@ -0,0 +1,51 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<?xml-stylesheet href="urn:x-suse:xslt:profiling:docbook50-profile.xsl"
+ type="text/xml"
+ title="Profiling step"?>
+<!DOCTYPE book
+[
+  <!ENTITY % entities SYSTEM "entity-decl.ent">
+    %entities;
+]>
+
+<!-- ##################################################################### -->
+<!-- SLE Container Guide                                                  -->
+<!-- ##################################################################### -->
+<?provo dirname="upgrade/"?>
+<book version="5.0" xml:lang="en" xml:id="book-sles-container"
+ xmlns="http://docbook.org/ns/docbook"
+ xmlns:xi="http://www.w3.org/2001/XInclude"
+ xmlns:xlink="http://www.w3.org/1999/xlink">
+ <info>
+  <title>Container Guide</title><productname>&productname;</productname>
+  <productnumber>&productnumber;</productnumber><date>
+   <?dbtimestamp format="B d, Y"?></date>
+  <xi:include href="common_copyright_gfdl.xml"/>
+  <abstract>
+
+    <para>
+     This document is a work in progress. The content in this document is
+     subject to change without notice.
+    </para>
+
+   <para>
+    &abstract_upgrade;
+   </para>
+  </abstract>
+  <dm:docmanager xmlns:dm="urn:x-suse:ns:docmanager">
+   <dm:bugtracker>
+    <dm:assignee>fs@suse.com</dm:assignee>
```

Comment:
Is this correct? (If you integrate this into MAIN.SLEDS.xml, you can probably remove the entire dm:bugtracker element from here -- and just use the fact that this information is inherited.)